sunpy/map/map_factory.py

```diff
@@ -350,12 +350,33 @@ def _is_url(arg):
 
 
 def _is_path(arg):
```

Comment:
I think the naming of this function is now misleading. I initially thought the `_parse_args` code above was broken because a glob is not a valid path (though it can be coerced into a `pathlib.Path` object).